colin-api/src/colin_api/models/filing.py

```diff
@@ -26,16 +26,17 @@
 from flask import current_app
 from registry_schemas.utils import get_schema
 
-from colin_api.exceptions import (  # noqa: I001
-    FilingNotFoundException,  # noqa: I001
-    GenericException,  # noqa: I001
-    InvalidFilingTypeException,  # noqa: I001
-    OfficeNotFoundException,  # noqa: I001
-    PartiesNotFoundException,  # noqa: I001
-)  # noqa: I001
+from colin_api.exceptions import GenericException  # noqa: I001
```

Comment:
why was this reformatted, the prior format is our preference.

Reply:
I used isort. Shouldn't I use isort?